### ISSUE TITLE
v11.2.9 NX   ERROR  Could not find Nx modules in this workspace.

### ISSUE THREAD

**Issue by adrian-ub** [nrwl/nx#4691]
## Current Behavior

When try create new project with `npx create-nx-workspace` or `yarn create nx-workspace` and with option --packageManager=yarn

## Expected Behavior

Create a new project successfully as before
I created a new project three days ago and had no error

## Steps to Reproduce

npx create-nx-workspace --packageManager=yarn
? Workspace name (e.g., org name)     adrianub
? What to create in the new workspace empty
? Use Nx Cloud? (It's free and doesn't require registration.) No

This issue may not be prioritized if details are not provided to help us reproduce the issue.

### Failure Logs

Creating a sandbox with Nx...
[#############################################################################################################] 421/421
new adrianub --packageManager=yarn --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace
yarn run v1.22.10
$ C:\Users\adrianub\AppData\Local\Temp\tmp-3496ulVxPfwInEk1\node_modules\.bin\tao new adrianub --packageManager=yarn --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace/collection.json --cli=nx ^"--nxWorkspaceRoot=\^"G:/projects/adrianub\^"^"
CREATE adrianub/nx.json
CREATE adrianub/package.json
CREATE adrianub/README.md
CREATE adrianub/tools/generators/.gitkeep
CREATE adrianub/tools/tsconfig.tools.json
CREATE adrianub/tsconfig.base.json
CREATE adrianub/.editorconfig
CREATE adrianub/.gitignore
CREATE adrianub/.prettierignore
CREATE adrianub/.vscode/extensions.json
CREATE adrianub/workspace.json
CREATE adrianub/.prettierrc
CREATE adrianub/apps/.gitkeep
CREATE adrianub/libs/.gitkeep
$ nx g @nrwl/workspace:preset --name= --style=css --linter=eslint --npmScope=adrianub --preset=empty --cli=nx --interactive=false

>  NX   ERROR  Could not find Nx modules in this workspace.

  Have you run npm/yarn install?

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Workspace creation failed, see above.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
(node:3496) UnhandledPromiseRejectionWarning: Error: Command failed: yarn tao new adrianub --packageManager=yarn --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace/collection.json --cli=nx --nxWorkspaceRoot=\"G:/projects/adrianub\"
    at checkExecSyncError (child_process.js:616:11)
    at Object.execSync (child_process.js:652:15)
    at createApp (C:\Users\adrianub\AppData\Local\Yarn\Data\global\node_modules\create-nx-workspace\bin\create-nx-workspace.js:380:21)
    at C:\Users\adrianub\AppData\Local\Yarn\Data\global\node_modules\create-nx-workspace\bin\create-nx-workspace.js:97:29
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:3496) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:3496) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

### Environment

Windows 10 Pro
node: v14.15.1
yarn: 1.22.10
npm: 6.14.8

**Comment by FrozenPandaz** [comment]
Version `11.2.10` includes some changes which might fix this. I can't tell which version is used here. Can you please retry with the latest version?

**Comment by adrian-ub** [comment]
The version when I wrote the issue is `11.2.9`

Now I have tried with the mentioned version `11.2.10` and this is the error

```
npx create-nx-workspace@latest prueba --packageManager=yarn
npx: installed 240 in 59.624s
? What to create in the new workspace empty             [an empty workspace with a layout that works best for building a
pps]
? Use Nx Cloud? (It's free and doesn't require registration.) No
Creating a sandbox with Nx...
[#############################################################################################################] 421/421
new prueba --packageManager=yarn --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace
yarn run v1.22.10
$ C:\Users\adrianub\AppData\Local\Temp\tmp-1376TZUVOZBrPU6Q\node_modules\.bin\tao new prueba --packageManager=yarn --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace/collection.json --cli=nx ^"--nxWorkspaceRoot=\^"G:/projects/adrianub\^"^"
CREATE prueba/nx.json
CREATE prueba/package.json
CREATE prueba/README.md
CREATE prueba/tools/generators/.gitkeep
CREATE prueba/tools/tsconfig.tools.json
CREATE prueba/tsconfig.base.json
CREATE prueba/.editorconfig
CREATE prueba/.gitignore
CREATE prueba/.prettierignore
CREATE prueba/.vscode/extensions.json
CREATE prueba/workspace.json
CREATE prueba/.prettierrc
CREATE prueba/apps/.gitkeep
CREATE prueba/libs/.gitkeep
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
(node:1376) UnhandledPromiseRejectionWarning: Error: Command failed: yarn tao new prueba --packageManager=yarn --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace/collection.json --cli=nx --nxWorkspaceRoot=\"G:/projects/adrianub\"
    at checkExecSyncError (child_process.js:616:11)
    at Object.execSync (child_process.js:652:15)
    at createApp (C:\Users\adrianub\AppData\Roaming\npm-cache\_npx\1376\node_modules\create-nx-workspace\bin\create-nx-workspace.js:380:21)
    at C:\Users\adrianub\AppData\Roaming\npm-cache\_npx\1376\node_modules\create-nx-workspace\bin\create-nx-workspace.js:97:29
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1376) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by bekos** [comment]
@adrian-ub we just released v11.2.12 which should contain the fix for this: 57c6bacfb4c33aba5abe13a742791b5b315da528. Can you please try again and let us know if this works now as expected?

**Comment by adrian-ub** [comment]
@bekos  Works!! thanks for fix this
